src/kubernetes_cleanup_jobs/kubernetes_cleanup_jobs/config.py

```diff
@@ -0,0 +1,75 @@
+import os
+import argparse
+import logging
+
+import kubernetes_cleanup_jobs.context
+from lib.logging import setup_logging
+
+class AppConfig:
+  def __init__(self, cmdline_args, env):
+    ''' set app-wide configs and initialize the app '''
+    nhours, incomplete, namespace, force, loglevel = (
+      int(cmdline_args.nhours),
+      cmdline_args.incomplete,
+      cmdline_args.namespace,
+      cmdline_args.force,
+      cmdline_args.loglevel,
+    )
+
+    context = env
+
+    self.nhours = nhours
+    self.incomplete = incomplete
+    self.namespace = namespace
+    self.force = force
+    self.context = context
+
+    self._init_app(loglevel)
+
+  def _init_app(self, loglevel):
+    ''' initialize the app '''
+    setup_logging(eval('logging.' + loglevel))
+
+def parse_args():
+  ''' parse command line args '''
+  parser = argparse.ArgumentParser(
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter
+  )
+  parser.add_argument(
+    'nhours',
+    help='delete jobs older than n hours'
+  )
+  parser.add_argument(
+    '--incomplete',
```

Comment:
kept the same flag as [`manage.py cleanup_jobs`](https://github.com/artsy/substance/blob/main/manage.py#L382) to target jobs that are not _completed_